core/headers.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+package core;
+
+
+option go_package = "Rinnegan/proto-generated/core";
+
+// Request header message
+message RequestHeader {
+    string app_id = 1;       // Unique identifier for your app
+    AppVersion app_version = 2;  // Version of the app
+    string secure_token = 3; // Token to authenticate the request, Planning to take it as JWT Token
+    Platform platform = 4;     // Platform making the request (e.g., iOS, Android, Web)
+    google.protobuf.Timestamp timestamp = 5;    // Timestamp of the request
+}
+
+// Response header message
+message ResponseHeader {
+    string response_code = 1; // Response code (e.g., 200 for success)
```

Comment:
How is this related to the enum below for ResponseHeader  ?

Reply:
Sorry missed using the enum here, will change